packages/app/src/hooks/useFaucet.ts

```diff
@@ -45,7 +46,7 @@ export function useFaucet(opts: UseFaucetOpts = {}) {
       onSuccess: () => {
         // Navigate to assets page to show new cons
         // https:// github.com/FuelLabs/swayswap-demo/issues/40
-        refreshBalances();
+        balances.refetch();
```

Comment:
@luizstacio changed this couple pull requests earlier, why did you change it again?

Reply:
just for the record, we talked about this and changed other places in the application as well to use `balances.refetch();` instead of `refreshBalances();`